hub/server.go

```diff
@@ -104,17 +105,17 @@ func (h *Hub) chainHandlers(acmeHosts []string) http.Handler {
 
 	r := mux.NewRouter()
 
+	if h.config.GetBool("metrics") {
+		r.Handle("/metrics", promhttp.Handler()).Methods("GET")
```

Comment:
Am I missing something or `hub.Metrics` is not used at all? I'm 100% fine with this simpler approach by the way (I prefer it). I'm just wondering if all the boilerplate code can't be removed (if used at all)?

Reply:
That's a mistake. In this PR, the `hub.Metrics` is not used.

The idea is to be flexible about the metrics system we can use.
But it's overkill, I will remove all the boilerplate, it's useless.